src/Sulu/Bundle/MediaBundle/Entity/FileVersionMeta.php

```diff
@@ -29,19 +34,43 @@ class FileVersionMeta
     /**
      * @var string
      */
-    private $locale;
+    private $copyright;
 
     /**
-     * @var int
+     * @var string
      */
-    private $id;
+    private $locale;
 
     /**
-     * @var \Sulu\Bundle\MediaBundle\Entity\FileVersion
+     * @var FileVersion
      */
     private $fileVersion;
 
     /**
+     * Get id.
+     *
+     * @return int
+     */
+    public function getId()
+    {
+        return $this->id;
+    }
+
+    /**
+     * Set id.
+     *
+     * @param int
+     *
+     * @return int
+     */
+    public function setId($id)
```

Comment:
Is this really necessary? `setId` should actually never be offered, since the database is setting the IDs for us.